autoload/android.vim

```diff
@@ -248,9 +248,10 @@ function! android#setClassPath()
     return
   endif
 
-  if exists(":JCstart")
```

Comment:
@sooth-sayer It should, would you mind opening an issue in the JavaComplete2 repo? I can hopefully fix it from there once I have more information